pywbem/mof_compiler.py

```diff
@@ -2363,8 +2363,8 @@ def compile_string(self, mof, ns, filename=None):
 
         Raises:
 
+          IOError: MOF include file not found.
```

Comment:
I think we both caught this one.  I also fixed it in #1138 pr for mofcompile-str.  Note that I also fixed it for the compile_string since that could also catch the exception not that it can compile pragmas

Reply:
Ok. The merge will sort that out.